printer_mods/See1807/Voron V0 Exhaust and LED Mod/README.md

```diff
@@ -0,0 +1,38 @@
+# Voron V0 Exhaust and LED Mod
+## _No stink, good light_
+
+Since the lack of suction on the VO bothered me and, in my opinion, printers should always have light, I developed this mod.
+
+- no more stink
+- bright, even light
+- Extraction even without LED lighting
+
+## BOM
+## _Exhaust_
+
+- 1 Heatset Insert M3
+- 1 Screw M3x8 BHCS
+
+## _LED Light_
+
+- 4 Heatset Insert M3
+- 4 Screw M3x12 BHCS
+- 4 Screw M3x10 BHCS
+- 2 x 10cm LED Stripe 24V 60LED7Meter
+
+## Build
+
+## _Exhaust_
+
+The exhaust can be exchanged "plug & play" for the rear top head panel. A heatset insert is inserted into the panel to attach the cover. The exhaust carries 30mm filter material. It is recommended to use both activated carbon filters and HEPA air filter mats. The lid is closed with an M3 * 8mm screw.
+
+There are 2 back panels in the folder. One with holes to pull the cables of the LED lamps through and one without, if you don't want to install LED lamps.
+
+## _LED Light_
+
+The LED mountigplate is provided with two heatset inserts at the outer holes. The two inner holes are used for mounting on the head. to do this, the original M3x10 screws are exchanged for M3x12.
+
+The LED strips are glued into the LED carrier after 24 AWG cables have been soldered to them. there is a separate carrier for each side with a recess to pull the cables out of the lamp.
+Finally, the carriers are screwed between the LED cover and the mounting plate with two M3x10mm screws each.
+
+A 12mm 24V switch can be used as a switch. I have included an adapted skirt for this.
```

Comment:
`Skirt 12mm switch.stl` is upside down.

Also, please link to an example switch that could be used here.